ts/__e2e__/login.e2e.ts

```diff
@@ -0,0 +1,18 @@
+import adapter from "detox/runners/jest/adapter";
+import { loginWithSPID } from "./utils";
+
+describe("User Login using SPID", () => {
+  beforeEach(async () => {
+    await adapter.beforeEach();
+  });
+
+  beforeAll(async () => {
+    await device.launchApp({ permissions: { notifications: "YES" } });
```

Comment:
This shouldn't be needed if we have it in [init.js](https://github.com/pagopa/io-app/pull/3482/files#diff-61c2c6be23549b2e77932b101b1d0c2adaa2803228307e1499cabaec750c599aR14) already?

Reply:
Yes you're right, is an unnecessary step, removed in https://github.com/pagopa/io-app/pull/3482/commits/0ce7d2a2f13df311befa95be49e9ab427a5592d6!